src/styles/variables.css

```diff
@@ -1,4 +1,335 @@
+  --global-line-height: 1.42857143;
+  --unit: 13px;
+  --distancer-small: 5px;
+  --distancer: 30px;
+  --background-content: #F3F3F4;
+  --color-background: #FFF;
+  --color-black: #676A6C;
+  --color-primary: #3DCCB4;
+  --color-warning: var(--palette-orange-300);
+  --color-leer-blue: #2DACE3;
+  --color-leer-yellow: #F49606;
+  --color-gray-light: #F3F3F4;
+  --color-gray-medium: #888A8C;
+  --color-gray: #8A8C8E;
+  --color-petrol: #293846;
+  --color-petrol-light: #2F4050;
+  --color-purple: #766D91;
+  --color-petrol-dark: color(var(--color-petrol) shade(5%));
+  --color-primary-dark: color(var(--color-primary) shade(50%));
+  --font-size: 13px;
+  --font-size-tiny: 11px;
+  --font-size-small: 12px;
+  --font-size-normal: var(--font-size);
+  --font-size-medium: 14px;
+  --font-size-big: 18px;
+  --font-weight-thin: 300;
+  --font-weight-normal: 400;
+  --font-weight-semi-bold: 500;
+  --font-weight-bold: 600;
+  --font-weight-bolder: 700;
+  --color-dark-yellow: #F5A623;
+  --color-dark-red: #D0021B;
+  --color-subject-DMT: #FE6B6B;
+  --color-subject-RW: #5CB8E6;
+  --color-subject-SP: #FFC800;
+  --color-subject-TBL: #A48AD5;
+  --color-subject-WST: #B8D916;
+  --color-subject-RvK: #5CB8E6;
+  --color-subject-TvK: #B8D916;
+  --h3-padding: calc(var(--distancer) / 3);
+  --h3-border-width: 2px;
+  --h3-height: var(--distancer);
+  --h3-content-height: var(--distancer);
+  --standard-increment-mobile: calc(5.6 * var(--unit));
+  --standard-increment-desktop: calc(6.4 * var(--unit));
+}
+
+/* Orientation */
+@custom-media --portrait (orientation: portrait);
+@custom-media --landscape (orientation: landscape);
+
+/* Devices (defined by max width) */
+@custom-media --xxs-viewport (max-width: 480px);
+@custom-media --xs-viewport (max-width: 600px);
+@custom-media --sm-tablet-viewport (max-width: 720px);
+@custom-media --sm-viewport (max-width: 840px);
+@custom-media --md-viewport (max-width: 960px);
+@custom-media --lg-tablet-viewport (max-width: 1024px);
+@custom-media --lg-viewport (max-width: 1280px);
+@custom-media --xl-viewport (max-width: 1440px);
+@custom-media --xxl-viewport (max-width: 1600px);
+@custom-media --xxxl-viewport (max-width: 1920px);
+
+/* Devices (defined by min-width) */
+@custom-media --larger-than-xxs-viewport (min-width: 480px);
+@custom-media --larger-than-xs-viewport (min-width: 600px);
+@custom-media --larger-than-sm-tablet-viewport (min-width: 720px);
+@custom-media --larger-than-sm-viewport (min-width: 840px);
+@custom-media --larger-than-md-viewport (min-width: 960px);
+@custom-media --larger-than-lg-tablet-viewport (min-width: 1024px);
+@custom-media --larger-than-lg-viewport (min-width: 1280px);
+@custom-media --larger-than-xl-viewport (min-width: 1440px);
+@custom-media --larger-than-xxl-viewport (min-width: 1600px);
+@custom-media --larger-than-xxxl-viewport (min-width: 1920px);
+
+/* scss variables for @each etc */
+
+/* $list-subjects: DMT, RW, SP, TBL, WST, RvK, TvK; */
+
+.red {
```

Comment:
Remove all the helper classes. We'll add them as we go. The idea is to keep this as slim as posible with only thing we're using.

Regarding the variables, the same keep only what you're currently using in the components.